src/components/campaignguide/Narrator.tsx

```diff
@@ -0,0 +1,545 @@
+import EncounterIcon from "@icons/EncounterIcon";
```

Comment:
If you could use single quote strings on the import statements, that would be great.

I also sort all the system packages to a top block, followed by all the imports from this repository.